soh/src/overlays/actors/ovl_En_Shopnuts/z_en_shopnuts.c

```diff
@@ -69,6 +69,15 @@ void EnShopnuts_Init(Actor* thisx, GlobalContext* globalCtx) {
     CollisionCheck_SetInfo(&this->actor.colChkInfo, NULL, &sColChkInfoInit);
     Collider_UpdateCylinder(&this->actor, &this->collider);
 
+    if (gSaveContext.n64ddFlag) {
+        s16 respawnData = gSaveContext.respawn[RESPAWN_MODE_RETURN].data & ((1 << 8) - 1);
+        ScrubIdentity scrubIdentity = Randomizer_IdentifyScrub(globalCtx->sceneNum, this->actor.params, respawnData);
+
+        if (scrubIdentity.isShuffled && gSaveContext.scrubsPurchased[scrubIdentity.scrubId] == 1) {
+            Actor_Kill(&this->actor);
```

Comment:
do we want to actor_kill here? how does 64rando handle this? i'd think we'd want to have it so after getting the mysterious item you could get the vanilla item from the scrub but i'm not sure

Reply:
N64 makes them disappear. I feel like it'd be more confusing to have them there after you got their item. Also consistent with Shopsanity where the major items are sold out rather than being replaced with vanilla shop items.

Reply:
coolcool, sounds like actor_kill works then